### PR TITLE
Py3 BigQuery Fixes for Sync process and Import Organizations

### PR DESCRIPTION
**Pull request by robabram**
Fixes a minor BigQuery sync issue.
Fixes the `localhost` BigQuery sync issue after running `import_organizations.sh`.
Use the DB connection string created by the cli tools when they are run.  Otherwise we only connect to our localhost DB when running cli tools.

### CODE REVIEW

**Comment by Seeker1911** [comment]
looks good so far.

File: ci/test.sh [hunk 1]
```diff
@@ -31,10 +31,10 @@ cd rdr_service
 
 # Shouldn't need this anymore
 # ./tools/install_config.sh --config=config/config_dev.json --update
-./tools/setup_local_database.sh --db_user root --db_name rdr
+# ./tools/setup_local_database.sh --db_user root --db_name rdr
```
Comment (Seeker1911) [inline ci/test.sh]: dont we still need this ?

Comment (robabram) [reply ci/test.sh]: If the unit tests all use the 9306 db instance, we don't.  I believe commenting it out will let us catch tests that try to use the 3306 db instance.  Using the 3306 db instance would indicate that something is wrong that needs correcting.

Comment (Seeker1911) [reply ci/test.sh]: ok, isnt this how CI runs its tests though ?

Comment (robabram) [reply ci/test.sh]: It runs them the same way as we do, it spins up a db instance on port 9306.

**Pull request merged**